register/views.py

```diff
@@ -60,7 +60,7 @@ def register_user(request, user_type):
                         "token": account_activation_token.make_token(f),
                     },
                 )
-                to_supervisor = form.cleaned_data["supervisor_email"]
+                to_supervisor = f.school.school_email
```

Comment:
Having this value with the live version would shoot up an email from our mail-server to the school's actual email value. This might be a cause of concern since we don't have any linking setup with any of the schools.